database/migrations/2017_07_08_202506_add_user_bio_column.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
+
+class AddUserBioColumn extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function ($table) {
+            $table->string('bio', 140)->nullable();
```

Comment:
Make this non-nullable. Increase this to 160 characters. Add a default value of an empty string.